openpathsampling/tests/test_storage.py

```diff
@@ -68,7 +65,7 @@ def test_create_storage(self):
         store.close()
 
     def test_stored_topology(self):
-        raise SkipTest
+        pytest.skip()
```

Comment:
what is the reason that this test is skipped?

Reply:
looks like it is a test that @jhprinz started 8 years ago and never quite finished? It tries to load a template topology, but (1) never stores it; (2) netcdfplus storages don't seem to have the `template` attribute. Leaving as-is for now, it'll probably get yanked as part of a larger cleanup in the future.

Reply:
👍 just wanted to make sure it was still a conscious decision